docs/workflows/translation.md

```diff
@@ -0,0 +1,224 @@
+# Translation
```

Comment:
Maybe this page should stay in the normal documentation and refer to it.
- https://docs.mapbender.org/develop/en/architecture/translation.html
the developer docu could containt some extended info like the bin/console command to generate the translation